samples/apps/org.wso2.carbon.uuf.sample.features-app/src/main/pages/helpers/built-in-block-helpers/with.hbs

```diff
@@ -0,0 +1,71 @@
+{{layout "org.wso2.carbon.uuf.sample.foundation.main"}}
+{{title "with Block Helper Sample | " @config.appName}}
```

Comment:
Title should start with Capital letters and also use a meaningful name for the page title